virtual_machine/windows/variables.tf

```diff
@@ -30,3 +30,12 @@ variable "vm_size" {
 variable "public_ip_allocation" {
   default = "dynamic"
 }
+
+variable "storage_image_reference" {
```

Comment:
This is a good start, but can we have this extended as a hash with different image types to select from? So for the calling module we can just specify the type and have the rest of the storage image reference attributes determined that way?

Reply:
Updated the keys are:
windows_server or sql_server 

  server_type      = "sql_server"